containers/bun/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# [Choice] Debian OS version: bullseye, buster
+ARG VARIANT=bullseye
+FROM --platform=linux/amd64 mcr.microsoft.com/vscode/devcontainers/base:0-${VARIANT}
+
+ENV BUN_INSTALL=/bun
+RUN mkdir -p /bun \
+    && curl -fsSL https://bun.sh/install | sh \
```

Comment:
I think `sh` would start to fail since [bun.sh](https://bun.sh/) changed the curl command to `bash`.
Refer the working devcontainer from [@alertbox/vscode-remote-try-bun](https://github.com/alertbox/vscode-remote-try-bun)